test/telnyx/messaging_short_code_test.rb

```diff
@@ -1,33 +1,33 @@
 # frozen_string_literal: true
 
-require ::File.expand_path("../../test_helper", __FILE__)
+# require ::File.expand_path("../../test_helper", __FILE__)
```

Comment:
Do we need this test file? If not lets get rid of it instead of keeping it commented out.